### PR TITLE
[WAZO-2586] use implementer instead of implements

### PR DESCRIPTION
**Pull request by bloom1**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,10 +4,11 @@ https://github.com/wazo-platform/wazo-auth-client/archive/master.zip
...
 jinja2==2.10
+markupsafe==1.1.0  # from jinja2
 pyopenssl==19.0.0
 pyyaml==3.13
 requests==2.21.0
 six==1.12.0
 stevedore==1.29.0
 twisted==18.9.0
-zope.interface==4.3.2
+zope.interface==4.7.2
```
Comment (pc-m) [inline requirements.txt:14]: Is this version required? It's not available in Debian at the moment

Comment (bloom1) [reply requirements.txt]: For this PR I think we can avoid changing the version, but I think I was not able to make it work using Python3 with version 4.3.2

Comment (sopelj) [reply requirements.txt]: The changelog for zope says `4.6.0` "Adds support for Python 3.7`. So, I would assume we need at least that.
But debian does ship with zope 4.3.2 and Python 3.7.... which is confusing.

**Comment by pc-m** [comment]
The change is pretty trivial to review. I don't see anything shocking. What should I do to try some of these changes?

**Comment by pc-m** [comment]
> Build failed.
> 
> * [wazo-tox-py27 ](https://zuul.wazo.community/zuul/t/local/build/8416d6fdff8f425eb01d1536b4b6516c) : SUCCESS in 4m 09s
> * [wazo-tox-py37 ](https://zuul.wazo.community/zuul/t/local/build/aedea70fd9ea417ca4d9ead824156740) : FAILURE in 2m 54s
> * [wazo-tox-integration ](https://zuul.wazo.community/zuul/t/local/build/bcf2d65fd9ea4c83a0500ef47a949c6f) : SUCCESS in 6m 26s
> * [debian-packaging ](https://zuul.wazo.community/zuul/t/local/build/723ef6b8577749f2993d2168a33b1fb5) : SUCCESS in 3m 37s

I broke the test by getting Zuul to execute the new `py37` target.  Any idea how to fix this?

**Comment by bloom1** [comment]
@pc-m yes, don't run it in python3. This is not the branch that makes it compatible with python3 :smile:

**Comment by pc-m** [comment]
> @pc-m yes, don't run it in python3. This is not the branch that makes it compatible with python3 smile

ok, reverted

**Comment by bloom1** [comment]
> The change is pretty trivial to review. I don't see anything shocking. What should I do to try some of these changes?

Nothing special. Only restart provd and see if any traceback pops up.

**Review by pc-m — APPROVED** [review]

**Comment by bloom1** [comment]
@pc-m the PR needed a rebase, sorry.

**Review by pc-m — APPROVED** [review]

**Pull request merged**